pkg/skaffold/docker/syncmap.go

```diff
@@ -75,42 +74,39 @@ func walkWorkspaceWithDestinations(workspace string, excludes []string, fts []fr
 
 		switch mode := fi.Mode(); {
 		case mode.IsDir():
-			if err := godirwalk.Walk(absFrom, &godirwalk.Options{
-				Unsorted: true,
-				Callback: func(fpath string, info *godirwalk.Dirent) error {
-					if fpath == absFrom {
-						return nil
-					}
+			if err := walk.From(absFrom).Unsorted().Do(func(fpath string, info walk.Dirent) error {
+				if fpath == absFrom {
```

Comment:
It feels like this excluding logic should be provided to the walker, something like:
```
walk.From(absFrom).Unsorted().Excluding(excluder).Do(...)
```
WDYT?

Reply:
It should indeed. I've just got started working on the Excluder. When it's more clearly defined, I'll add it to the walk API